x-pack/plugins/security_solution/server/lib/risk_engine/risk_engine_data_client.ts

```diff
@@ -199,9 +196,9 @@ export class RiskEngineDataClient {
     namespace = DEFAULT_NAMESPACE_STRING,
   }: InitializeRiskEngineResourcesOpts) {
     try {
-      const esClient = await this.options.elasticsearchClientPromise;
+      const esClient = this.options.esClient;
 
-      const indexPatterns = getIndexPattern(namespace);
+      const indexPatterns = getIndexPatternDataStream(namespace);
```

Comment:
Nit: I find this function name, and its response, very confusing. It's also got a lot of overlap/interop with `getRiskScoreLatestindex`, now, too. Could we maybe rename this to something like `getRiskScoreResourceOptions`, which returns e.g. `templateName`, `templatePattern`, etc. so their uses are more clear?

I know that I reused this function somewhere myself, so I apologize if I made it worse, but whenever I see this function I always have to dig through the definition and usage to understand it, which is never a good sign.